lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeAvroDefinitionGenerator.java

```diff
@@ -19,14 +19,20 @@
 
 package org.apache.reef.runtime.multi.client;
 
+import org.apache.reef.annotations.Unstable;
 import org.apache.reef.runtime.multi.utils.avro.MultiRuntimeDefinition;
+import org.apache.reef.tang.annotations.DefaultImplementation;
 
 import java.net.URI;
 
 /**
  * Defines a contract for a multi runtime definition generator.
+ * The multi runtime avro definition contains all the needed information to instantiate
+ * runtimes inside runtime host.
  */
-public interface MultiRuntimeDefinitionGenerator {
+@DefaultImplementation(MultiRuntimeAvroDefinitionGeneratorImpl.class)
+@Unstable
+public interface MultiRuntimeAvroDefinitionGenerator {
```

Comment:
@afchung @markusweimer Actually I don't see any reason Adding Avro to the class name, we already have that in the namespace. WDYT? If we agree I will remove Avro from the names.